### DISCUSSION TITLE
Lazer stuck on black screen

### DISCUSSION THREAD

**Discussion by RaoraRare** [ppy/osu#22709] [Q&A] [answered]
### Type

Other

### Bug description

Lazer stays on black screen after start up

### Version

 2023.207.0 

### Logs

[database.log](https://github.com/ppy/osu/files/10788358/database.log)
[input.log](https://github.com/ppy/osu/files/10788359/input.log)
[network.log](https://github.com/ppy/osu/files/10788360/network.log)
[performance.log](https://github.com/ppy/osu/files/10788361/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/10788362/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/10788363/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/10788364/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/10788365/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10788366/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10788367/updater.log)

**Comment by bdach** [comment, marked as answer]
Can you attach a video of this? Does the audio play at all?

**Reply by RaoraRare** [reply]
Sounds works but that is it

**Reply by bdach** [reply]
Not sure then, sorry. Can you try checking if there are graphics driver updates for your hardware?

**Reply by RaoraRare** [reply]
Fixed updated graphics drivers
